CHANGELOG-3.8.md

```diff
@@ -694,6 +694,12 @@ initialization. See `InitializationData.pluginFactories`.
 
 ## MATLAB Changes
 
+- Changed the mapping for Slice dictionaries to MATLAB. A Slice dictionary now always maps to a MATLAB dictionary; the
```

Comment:
This first sentence is confusing. I reads like Slice dictionaries are not mapped to a type called "MATLAB". I would just remove the "to MATLAB"

Reply:
Fixed as suggested.